components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/show-subscriber-status.jsp

```diff
@@ -186,14 +187,14 @@
                 if(dto != null && dto.getTimeInstance() != null){
         %>
         <tr>
-            <td><%=(new Date(Long.parseLong(dto.getTimeInstance()))).toString()%></td>
-            <td><% if(dto.getType() != null){%> <%=dto.getType()%><%}%></td>
-            <td><% if(dto.getUser() != null){%> <%=dto.getUser()%><%}%></td>
-            <td><% if(dto.getTarget() != null){%> <%=dto.getTarget()%><%}%></td>
-            <td><% if(dto.getTargetAction() != null){%> <%=dto.getTargetAction()%><%}%></td>
+            <td><%=(Encode.forHtml(new Date(Long.parseLong(dto.getTimeInstance()))).toString())%></td>
```

Comment:
This is type safe since converted to a date and date is converted back. Not necessary to encode